addons/binding/org.openhab.binding.vallox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,415 @@
+	</channel-type>
+
+	<channel-type id="CascadeAdjust" advanced="true">
+		<item-type>Switch</item-type>
+		<label>CascadeAdjust</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="MaxSpeedLimitMode" advanced="true">
+		<item-type>Switch</item-type>
+		<label>MaxSpeedLimitMode</label>
+		<description>Function of max speed limit (0 = with adjustment, 1 = always)</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="ServiceReminder" advanced="true">
+		<item-type>Number</item-type>
+		<label>ServiceReminder</label>
+		<description>Service Reminder in months; informs about the next maintenance alarm time: remaining months, descending.</description>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="PostHeatingOn" advanced="true">
+		<item-type>Switch</item-type>
+		<label>PostHeatingOn</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="DamperMotorPosition" advanced="true">
+		<item-type>Switch</item-type>
+		<label>DamperMotorPosition</label>
+		<description>Damper Motor Position - OFF=winter ON=season</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="FaultSignalRelayClosed" advanced="true">
+		<item-type>Switch</item-type>
+		<label>FaultSignalRelayClosed</label>
+		<description>fault signal relay closed - OFF=open ON=closed</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="SupplyFanOff" advanced="true">
+		<item-type>Switch</item-type>
+		<label>SupplyFanOff</label>
+		<description>supply fan off - OFF=on ON=off</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="PreHeatingOn" advanced="true">
+		<item-type>Switch</item-type>
+		<label>PreHeatingOn</label>
+		<description>pre-heating - OFF=off ON=on</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="ExhaustFanOff" advanced="true">
+		<item-type>Switch</item-type>
+		<label>ExhaustFanOff</label>
+		<description>exhaust-fan off - OFF=on ON=off</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="FirePlaceBoosterClosed" advanced="true">
+		<item-type>Switch</item-type>
+		<label>FirePlaceBoosterClosed</label>
+		<description>fireplace-booster closed - OFF=open ON=closed</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="IncommingCurrent" advanced="true">
+		<item-type>Number</item-type>
+		<label>IncommingCurrent</label>
+		<description>Current/Voltage in mA incomming on machine</description>
+		<state min="0" max="255" step="1" pattern="%d" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="LastErrorNumber" advanced="true">
+		<item-type>Number</item-type>
+		<label>LastErrorNumber</label>
+		<description>  5: Supply air temperature sensor fault; 6: Carbon dioxide alarm; 7: Outdoor air sensor fault;
```

Comment:
Note that you can use HTML elements and CDATA sections in the descriptions - this looks as if you'd want to put it in a bullet point list.